validation-api-java8.sig

```diff
@@ -135,7 +135,7 @@ CLSS public abstract interface !annotation java.lang.annotation.Target
 intf java.lang.annotation.Annotation
 meth public abstract java.lang.annotation.ElementType[] value()
 
-CLSS public abstract interface javax.validation.BootstrapConfiguration
+CLSS public abstract interface jakarta.validation.BootstrapConfiguration
```

Comment:
If this sig file was generated newly, could this be documented for later reference.